packages/mdc-textfield/_mixins.scss

```diff
@@ -14,13 +14,13 @@
 // limitations under the License.
 //
 
-@import "@material/line-ripple/mixins";
-@import "outline/mixins";
-@import "@material/floating-label/mixins";
-@import "helper-text/mixins";
-@import "./variables";
 @import "icon/mixins";
```

Comment:
1. All subpaths of the current directory should start with `./`
2. Imports should be alphabetized
3. "External" imports (`@material/foo`) should go at the top

E.g.:

```scss
@import "@material/floating-label/mixins";
@import "@material/line-ripple/mixins";
@import "@material/theme/mixins";
@import "./helper-text/mixins";
@import "./icon/variables";
@import "./outline/mixins";
@import "./variables";
```